### PR TITLE
Add PT-BR translations

### PR DESCRIPTION
**Pull request by arielalvesdutra**
- Add packages/playground-examples/copy/pt/TypeScript/Primitives/Any.ts
- Add packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts
- Add packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts

### CODE REVIEW

**Comment by arielalvesdutra** [comment]
Hi @khaosdoctor, @alvarocamillont, @danilofuchs e @orta!

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Any.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+// Any declara ao Typescript para confiar no seu código
+// como sendo seguro porque você sabe mais sobre ele, mesmo se isso
+// não seja estritamente verdadeiro. 
+// Por exemplo, esse código iria falhar:
+
+myObject.x.y.z;
+
+// Utilizar any prove a você a habilidade de escrever código próximo 
+// ao JavaScript original sem a segurança do tipos.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Any.ts]:
```suggestion
// Utilizar any provê a você a habilidade de escrever código próximo 
// ao JavaScript original sem a segurança do tipos.
```

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Any.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+// A chamada de swap é autorizada porque o argumento pode ser 
+// correspondente ao repor o primeiro any do par com um 
+// número e o segundo any com uma string.
+
+// Se tuplas são novas para você, veja: example:tuples
+
+// Unknown é um irmão do tipo any, se any é sobre dizer "Eu sei 
+// o que é melhor", então Unknown é sobre dizer "Eu não sei o que 
+// é o melhor, então você precisa dizer ao TS o tipo".
+// example: unknown-and-never
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Any.ts]:
```suggestion
// Unknown é um irmão do tipo any, any é como dizer "Eu sei 
// o que é melhor", já Unknown é como dizer "Eu não sei o que 
// é o melhor, então você precisa dizer ao TS o tipo".
// example:unknown-and-never
```

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
+// Tipo unions é uma forma de declarar que um objeto
+// pode ser mais de um tipo.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts]:
```suggestion
// União de tipos (type unions) é uma forma de declarar que um objeto
// pode ser mais de um tipo.
```

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+type StringOrNumber = string | number;
+type ProcessStates = "open" | "closed";
+type OddNumbersUnderTen = 1 | 3 | 5 | 7 | 9;
+type AMessyUnion = "hello" | 156 | { error: true };
+
+// Se o uso de "open" e "closed" vs string é
+// novo para você, verifique: example: literals
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts]:
```suggestion
// novo para você, verifique: example:literals
```

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+// Nos podemos misturar diferentes tipos em um union, e
+// o que estamos dizendo é que o valor é um destes tipos.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts]:
```suggestion
// Podemos misturar diferentes tipos em um union,
// dizendo que o valor é de um destes tipos.
```

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+// TypeScript irá então deixar você descobrir como
+// determinar qual valor poderá ser em tempo de execução.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts]:
```suggestion
// TypeScript irá então deixar para você o trabalho de determinar
// qual valor deve ser em tempo de execução.
```

File: packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+type WindowStates = "open" | "closed" | "minimized" | string;
+
+// Se você passar o mouse por cima, você poderá ver que o WindowStates
+// se transforma em uma string - não em um union. Isto é explicado em
+// example:type-widening-and-narrowing
+
+// Se um union é um 'OU', então um intersection é um 'É'.
+// Tipo intersection é tipo com dois tipos que se cruzam 
+// para criar um novo tipo. Ele permite a composição de tipos.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Primitives/Union and Intersection Types.ts]:
```suggestion
// Se um union é um 'OU', então um intersection é um 'E'.
// Interseção de tipos (type intersection) consiste de dois tipos que 
// se cruzam para criar um novo tipo. Ele permite a composição de tipos.
```

File: packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts [hunk 8]
```diff
@@ -0,0 +1,122 @@
+//// { order: 3, compiler: { strictNullChecks: true } }
+
+// Quando um tipo específico parece util na maioria 
+// das bases de código, eles são adicionados ao TypeScript e
+// se tornam disponíveis para todos, o que significa que 
+// você pode contar com a disponibilidade deles.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts]:
```suggestion
// Quando um tipo específico parece útil na maioria 
// das bases de código, eles são adicionados ao TypeScript e
// se tornam disponíveis para todos, o que significa que 
// você pode contar com a disponibilidade deles.
```

File: packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+// Pega um objeto e transforma suas propriedades para apenas 
+// leitura.
+
+type StickerFromAPI = Readonly<Sticker>;
+
+// Record<KeysFrom, Type>
+
+// Cria um tipo que usa a lista de propriedaes do
+// KeysFrom e da a eles o valor do tipo.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts]:
```suggestion
// Cria um tipo que usa a lista de propriedades do
// KeysFrom e dá a eles o valor do tipo.
```

File: packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+// Cria um tipo pegando o conjunto de propriedades Keys
+// do Type. Essencialmente uma lista de permissões para extrair 
+// informações de tipo de um tipo.
+
+type StickerSortPreview = Pick<Sticker, "name" | "updatedAt">;
+
+// Omit<Type, Keys>
+
+// Cria um tipo removendo o conjunto de propriedas Keys
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts]:
```suggestion
// Cria um tipo removendo o conjunto de propriedades Keys 
```

File: packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+// Extract<Type, MatchUnion>
+
+// Cria um tipo com as proprieades de Type que sobrepõe MatchUnion.
+
+type DynamicPages = Extract<NavigationPages, "home" | "stickers">;
+
+// NonNullable<Type>
+
+// Cria um tipo removendo null e undefined do conjunto de propriedades.
+// Util quando você tem uma checagem de validação.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/TypeScript/Type Primitives/Built-in Utility Types.ts]:
```suggestion
// Útil quando você tem uma checagem de validação.
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Great job!
I have some minor suggestions and found some typos

**Comment by arielalvesdutra** [comment]
Thank you @danilofuchs and sorry for the typos. I created a new commit with the corrections.

**Comment by orta** [comment]
LGTM

**Comment by arielalvesdutra** [comment]
Hi, for some reason, the PR hasn't been merged.

**Comment by orta** [comment]
LGTM twice, if not I'll manually merge it - I was testing the merging tool

**Comment by arielalvesdutra** [comment]
Maybe will be necessary a manually merge.

**Pull request merged**